docs/src/contribute/code-of-conduct.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Code of Conduct
+eleventyNavigation:
+    key: code of conduct
+    parent: contribute to eslint
+    title: Code of Conduct
+    order: 0
```

Comment:
note: i realize that it's not standard practice to have this item have order `0`. however i would like that we allow this, at least temporarily while doing other IA update PRs as to not create merge conflicts/confusion. if the reviewers' are okay with this approach, i'll create a separate PR for fixing the contribute to eslint navigation order after all the IA updates are done.

Reply:
The deploy preview confirms that there's no issue with `0` accidentally being treated as falsy, so sure, this makes sense to reduce merge conflicts :+1: